src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,29 @@
+package lotto.domain;
+
+
+import java.util.List;
+import lotto.constant.message.ExceptionMessage;
+
+import static lotto.constant.LottoNumber.MAX_COUNT;
+
+public class Lotto {
+    private final List<Integer> numbers;
+
+    public Lotto(List<Integer> numbers) {
+        validate(numbers);
+        this.numbers = numbers;
+    }
+
+    private void validate(List<Integer> numbers) {
+        if (numbers.size() != MAX_COUNT.getNumber()) {
+            throw new IllegalArgumentException(ExceptionMessage.ERROR.getMessage());
+        }
+        if (numbers.stream().distinct().count() != MAX_COUNT.getNumber()) {
```

Comment:
조건문에 사용되는 조건이 어느 정도 복잡해 진다면, 해당 조건에 사용할 메서드를 따로 추출하는 것도 가독성에 큰 도움이 됩니다!
```suggestion
        if (hasDuplicatedNumbers(numbers)) {
```

Reply:
지난 리뷰에서도 이와 비슷한 피드백을 받았지만 이해가 잘 안되서 적용을 못했는데 예시를 보여주셔서 이번 미션부터 적용해볼수 있을 것 같아요! 감사합니다☺️